reflekt-plugin/src/test/kotlin/org/jetbrains/reflekt/plugin/scripting/KotlinScriptRunnerTest.kt

```diff
@@ -14,12 +14,14 @@ class KotlinScriptRunnerTest {
     }
 
     @Test
+    @Suppress("ArrayPrimitive")
     fun scriptWithProperties() {
         val script = KotlinScriptRunner(
             properties = listOf("a" to Array::class, "b" to String::class),
             code = "a.size.toString() + b",
         )
         assertEquals("42",
+            // TODO: intArray is not applicable here?
             script.eval(listOf(arrayOf(1, 2, 3, 4), "2")),
```

Comment:
this test is failing when I use `intArray`

Reply:
Sorry for the LONG delay((

I have checked it, and it works. Probably, you did not replace the 20 code line
I mean:
```
    fun scriptWithProperties() {
        val script = KotlinScriptRunner(
            properties = listOf("a" to IntArray::class, "b" to String::class),
            code = "a.size.toString() + b",
        )
        assertEquals("42",
            script.eval(listOf(intArrayOf(1, 2, 3, 4), "2")),
        )
    }
```

Since if you change only script.eval(listOf(intArrayOf(1, 2, 3, 4), "2")), the test will expect Array::class as a property and will fail

Reply:
please, delete this todo